Add Plain theme, using only the default templates

mvz opened the pull request:
Looks rather ugly, but shows what the default templates look like.

hmallett commented:
I like this - the idea that there should be an unstyled theme that you can then use to build new themes on top of. I do wonder if that should be it's purpose, and that transforming it in to "a moderately nice looking theme" is undesirable, as we have bootstrap-2 for that.

mvz commented:
I added a sprinkle of css to make it at least not horrible to use, and a preview image. Also, I discovered it was impossible to have a theme consisting of only css since there was no way for Publify to load that css. I fixed that, so now pure-css themes are possible. Yay!
